charts/tidb-operator/templates/controller-manager-deployment.yaml

```diff
@@ -1,7 +1,8 @@
+{{- if .Values.controllerManager.create }}
```

Comment:
```suggestion
{{- if .Values.controllerManager.create | default true }}
```

Reply:
We should make sure that the value is true even if users use the previous values.yaml without this configuration.

Reply:
BTW, with `{{- if .Values.controllerManager.create }}`, did you try to set it to `false` and confirm that it's not installed?
It seems that set it to `false` will still install the controller-manager.

Reply:
The similar comments for scheduler.

Reply:
Yes, this is a upstream bug. Please ref: https://github.com/Masterminds/sprig/issues/111

Reply:
So, I use ``{{- if (hasKey .Values.controllerManager "create" | ternary .Values.controllerManager.create true) }}``` to workaround it.